workflow/lib/shared/initialize_paramsearch.py

```diff
@@ -0,0 +1,62 @@
+"""Initialize parameter searches."""
+
+from snakemake.utils import Paramspace
+import pandas as pd
+from itertools import product
+from pathlib import Path
+
+from lib.shared.file_utils import get_filename
+
+
+def initialize_segment_sbs_paramsearch(config):
+    """Initialize parameter search for sbs segmentation by setting up config structure.
+
+    Args:
+        config (dict): Configuration dictionary.
+
+    Returns:
+        dict: Updated configuration dictionary.
+    """
+    if config["sbs_process"].get("mode") != "segment_sbs_paramsearch":
+        return config
+
+    if "paramsearch" not in config["sbs_process"]:
+        # Set default parameter search ranges if not specified
+        base_nuclei = config["sbs_process"]["nuclei_diameter"]
+        base_cell = config["sbs_process"]["cell_diameter"]
+
+        config["sbs_process"]["paramsearch"] = {
+            "nuclei_diameter": [base_nuclei - 2, base_nuclei, base_nuclei + 2],
+            "cell_diameter": [base_cell - 2, base_cell, base_cell + 2],
+            "flow_threshold": [0.2, 0.4, 0.6],
+            "cellprob_threshold": [-4, -2, 0, 2, 4],
+        }
```

Comment:
Adding any kind of documentation for where these numbers came from would be helpful, even if its something like "Our tests indicate this is a good range to start with. May need to adjust for other data". Just so others know when they look at this code why we have these specific numbers.

Reply:
Added more details in the notebooks where this is explained in [commit](https://github.com/cheeseman-lab/brieflow/pull/23/commits/db3760b366f14068cb22fd010a7078982b1dd0b6)